roles/add-nexus-repo/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+nexus_url: localhost:8081
+nexus_protocol: http
```

Comment:
The "default" for the `config_nexus` role is `https` - can we make sure to align the defaults? My vote would be to default it to `https`.

Reply:
@oybed There is forthcoming work where it would be required to use `oc port-forward` in order to do configuration around the OAuth Proxy, and that would require `http` as the default. Does it make sense to change this to `https` for now and then override specifically for those cases, or should we leave it as `http` for broader cases where users may not have SSL enabled for everything?

Reply:
I think we should be consistent across all roles + IMHO we should always default to "secure" if we can, so I'd say default to `https` and override in the cases where it needs to be http.